### PR TITLE
add carousel

### PR DESCRIPTION
**Pull request by natnoppol**
image

- [ ] Add JS - function to make slider move between images when clicking previous/next buttons, make active/current image larger than the others
- [ ] Add onclick events on element buttons

- [ ] Add div class="mySlides" every elements img
- [ ] Add  <script src="/js/carousel.js"></script>

### CODE REVIEW

**Review by Kittypoda — APPROVED** [review]

File: pages/gallery-page.html [hunk 1]
```diff
@@ -448,5 +460,6 @@ <h1 class="text-2xl font-semibold mb-6">
       </section>
       <!-- Section 4 ends here -->
     </main>
+    <script src="/js/carousel.js"></script>
```
Comment (Kittypoda) [inline pages/gallery-page.html:463]: Hello! Is there any reason why the script is not in the head ?:)

**Review by Kittypoda — COMMENTED** [review]
Added a comment

**Review by Kittypoda — CHANGES_REQUESTED** [review]
Added a comment :)

File: pages/gallery-page.html (hunk 1 above)
Comment (natnoppol) [reply pages/gallery-page.html:463]: Hei, Avoid Render-Blocking: JavaScript can be render-blocking, meaning it prevents the browser from rendering other content until the script is downloaded and executed. Placing scripts at the bottom minimizes this issue by allowing the browser to render HTML content first.

Comment (Kittypoda) [reply pages/gallery-page.html:463]: This is stated in the rules: 

JavaScript 
Coding Standards 
Modules: 
Use files with the ‘.mjs’ extension, which includes ‘import’ and ‘export’ statements to provide better code organization and reusability. 

You might want to consider creating an msj file instead?

Skjermbilde 2024-04-11 154537

Comment (natnoppol) [reply pages/gallery-page.html:463]: Our team misunderstood, this code "Gallery-slider.mjs" is belong to one of our team, its not mine. For now you can close this pull request.

**Pull request closed without merging**